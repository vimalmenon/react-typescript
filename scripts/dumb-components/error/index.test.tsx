import "jsdom-global/register";
import * as React from "react";
import * as renderer from "react-test-renderer";
import Error from "./index";
import {mount} from "enzyme";

describe("Error component :", () => {
    const message = "test";

    test("Snapshot renders :", () => {
        const component = renderer.create(<Error message={message}/>);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
    
    test("Renders the message passed as props :", () => {
        const wrapper = mount(<Error message={message}/>);
        expect(wrapper.find('span').text()).toEqual("test");
        
    });

});
