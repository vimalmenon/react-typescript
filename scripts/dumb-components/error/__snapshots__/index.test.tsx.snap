// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component : Snapshot renders : 1`] = `
<section
  className="sc-bdnylx eFcNAu"
>
  <span>
    test
  </span>
</section>
`;
